test/xml-data-processing.test.js

```diff
@@ -91,14 +91,14 @@ describe('XML processing', () => {
           const result = element.generatePayloadsExamples(payload, 'application/xml');
           const { value } = result[0];
           assert.include(value, '<schema>\n', 'has defualt name');
-          assert.include(value, '  <amf_inline_type_6>', 'has invalid properties');
```

Comment:
The point was to detect whether an xml tag name was generated properly, which should be `<amf_inline_type_6>`